src/main/channels/audio/GenerateAudiosTranscriptionChannel.ts

```
@@ -6,33 +6,63 @@ import { instanceToPlain } from 'class-transformer';
+        inputType: 'Audio',
+        sender: {
+          id: participantId,
+          type: 'user',
+        },
+      }); */
+
+      event.sender.send(
+        request.responseChannel,
+        instanceToPlain(transcription),
+      );
+    } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)